src/functionalTest/resources/scenarios/RIA-769-appeal-submission-summary.json

```diff
@@ -27,7 +27,7 @@
               "document": {
                 "document_url": "$/http.+\/documents/[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}/",
                 "document_binary_url": "$/http.+\/documents/[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}\/binary/",
-                "document_filename": "PA 11111 2019-González-appeal-form.PDF"
+                "document_filename": "PA 11111 2019-Gonzlez-appeal-form.PDF"
```

Comment:
Any reason why we changed the name?  Special character a problem?

Reply:
Yeah